### DISCUSSION TITLE
SWG percentage

### DISCUSSION THREAD

**Discussion by SethL** [aqualinkd/AqualinkD#165] [General]
Would it be possible to have more granular control over the SWG?  Currently when you modify the level it changes by 5% steps.  I usually run my SWG at 9% and will adjust up or down by single digits (I run high CYA and my SWG is a bit oversized for my pool).

**Comment by sfeakes** [comment]
There are very few variations of SWG / Control Panel and Keypads that support increasing less than 5% at a time, and the versions of SWG and Panel also come into play, so I just defaulted to 5%.
I can change to 1%, (using config option) but have no way to test (that I can think of at the moment).  
can you let me know the SWG, Panel version you have?  Also the keypad you are using for 1% increase?   Click the AqualinkD in the ribbon at the top of web UI will give you panel#

I’ll take a look in the current code base I’m working and see if I can add something that may work.

**Reply by SethL** [reply]
Sure and thanks! I'm using an aquapure SWG
![image](https://user-images.githubusercontent.com/2295127/161775429-1096a128-dfb5-4169-825b-148afaa04756.png)
For the 1% increase I am using iaqualink
